tests/test_end_to_end.py

```diff
@@ -73,11 +76,16 @@ def test_xarray_zarr_subpath(
                 target_root=tmp_target_url,
                 store_name="subpath",
                 combine_dims=pattern.combine_dim_keys,
+                consolidate_coords=consolidate_coords,
             )
         )
+    # TODO: This test needs to check if the consolidate_coords transform
+    # within StoreToZarr is consolidating the chunks of the coordinates
 
-    ds = xr.open_dataset(os.path.join(tmp_target_url, "subpath"), engine="zarr")
-    xr.testing.assert_equal(ds.load(), daily_xarray_dataset)
+    store = zarr.open(os.path.join(tmp_target_url, "subpath"))
+
+    # fails
+    assert netcdf_local_file_pattern_sequential.dims["time"] == store.time.chunks[0]
```

Comment:
@norlandrhagen what is the rationale behind this `assert`? It is not intuitive to me why these two values:

1. Would be `==` to one-another; or
2. How this would be evidence of coordinate consolidation

Additionally, in your view is the fact that this is currently failing evidence that the current implementation is wrong? Or that this `assert` is not checking the right thing?

Reply:
@cisaacstern I think it's the the assert isn't checking the right thing.